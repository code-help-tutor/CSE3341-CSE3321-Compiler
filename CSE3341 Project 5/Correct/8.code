procedure t8 and
	procedure A(r) is
		record n;
	begin
		if r < 1 then
			out(r);
		else
			n := new record[1];
			n[0] := r-1;
			begin A(n);
		end
	end
is
	record r;
begin
	r := new record[1];
	r[0] := 5;
	begin A(r);
end
