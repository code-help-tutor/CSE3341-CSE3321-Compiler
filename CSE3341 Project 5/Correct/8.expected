gc:1
gc:2
gc:3
gc:4
gc:5
gc:6
0
gc:5
gc:4
gc:3
gc:2
gc:1
gc:0
